classes/PodsUI.php

```diff
@@ -2823,383 +2823,401 @@ public function manage( $reorder = false ) {
+							$export_document_location = pods_slash(
+								pods_query_arg(
+									array(
+										'action_bulk' . $this->num => 'export',
+										'_wpnonce' => wp_create_nonce( 'pods-ui-action-bulk' ),
+									), self::$allowed, $this->exclusion()
+								)
+							);
+							?>
+							<div class="alignleft actions">
+								<input type="button" value="<?php echo esc_attr( sprintf( __( 'Export all %s', 'pods' ), $this->items ) ); ?>" class="button" onclick="document.location='<?php echo $export_document_location; ?>';" />
+							</div>
+							<?php
+						}//end if
 						?>
-						<div class="alignleft actions">
-							<input type="button" value="<?php echo esc_attr( sprintf( __( 'Export all %s', 'pods' ), $this->items ) ); ?>" class="button" onclick="document.location='<?php echo $export_document_location; ?>';" />
+						<br class="clear" />
 						</div>
 						<?php
+					} else {
+						?>
+						<br class="clear" />
+						<?php
 					}//end if
-					?>
-					<br class="clear" />
-					</div>
-					<?php
-				} else {
-					?>
-					<br class="clear" />
-					<?php
-				}//end if
-			?>
-			<div class="clear"></div>
-			<?php
-			if ( empty( $this->data ) && false !== $this->default_none && false === $this->search ) {
-				?>
-				<p><?php _e( 'Please use the search filter(s) above to display data', 'pods' ); ?>
-								<?php
-								if ( $this->export ) {
 				?>
-				, <?php _e( 'or click on an Export to download a full copy of the data', 'pods' ); ?><?php } ?>.</p>
+				<div class="clear"></div>
 				<?php
-			} else {
-				$this->table( $reorder );}
-			if ( ! empty( $this->data ) ) {
-				if ( true !== $reorder && ( false !== $this->pagination_total || false !== $this->pagination ) ) {
+				if ( empty( $this->data ) && false !== $this->default_none && false === $this->search ) {
 					?>
-					<div class="tablenav">
-						<div class="tablenav-pages<?php echo esc_attr( ( $this->limit < $this->total_found || 1 < $this->page ) ? '' : ' one-page' ); ?>">
-							<?php $this->pagination( 0 ); ?>
-							<br class="clear" />
-						</div>
-					</div>
+					<p><?php _e( 'Please use the search filter(s) above to display data', 'pods' ); ?>
+									<?php
+									if ( $this->export ) {
+					?>
+					, <?php _e( 'or click on an Export to download a full copy of the data', 'pods' ); ?><?php } ?>.</p>
 					<?php
+				} else {
+					$this->table( $reorder );}
+				if ( ! empty( $this->data ) ) {
+					if ( true !== $reorder && ( false !== $this->pagination_total || false !== $this->pagination ) ) {
+						?>
+						<div class="tablenav">
+							<div class="tablenav-pages<?php echo esc_attr( ( $this->limit < $this->total_found || 1 < $this->page ) ? '' : ' one-page' ); ?>">
+								<?php $this->pagination( 0 ); ?>
+								<br class="clear" />
+							</div>
+						</div>
+						<?php
+					}
 				}
-			}
 
-			?>
-		</form>
+				?>
+			</form>
+		</div>
+		<div class="pods-admin_friends-callout_container">
```

Comment:
@nicdford We probably want to add an action here instead, and hook into that action in PodsAdmin.php so we can show this only on specific pages we target. I can help on that part.